Multisig/Logic/Models/KeyInfo.swift

```diff
@@ -87,6 +91,10 @@ extension KeyInfo {
         }
     }
 
+    static func keysNeedBackup() -> [KeyInfo] {
```

Comment:
not sure if KeyInfo model should have such logic here. Maybe it could be placed in some repository class. 
Other suggestion is to rename keysNeedsBackup -> keysWithoutBackup